distributed/worker.py

```diff
@@ -1191,7 +1191,7 @@ def send_to_worker(self, address, msg):
             max_connections is not False
             and self.outgoing_current_count >= max_connections
         ):
-            logger.debug(
+            logger.warning(
```

Comment:
In https://github.com/dask/distributed/pull/3071#discussion_r330066652 we decided to leave this at debug level. Ideally these warnings would be throttled at a higher-level, but right now they'll show up frequently.

Reply:
Thanks for the quick feedback. I have to admit this was the one why I created this PR in the first place. It sounded too important to be not visible at all.